chapters/diffs-from-previous-editions.md

```diff
@@ -1,8 +1,8 @@
 # Annex I Differences from previous editions (Informative)
 
-# I.1 Differences between V2.2.1 and V2.2 <a name="I.1"></a>
```

Comment:
Instead of changing this section to talk about v2.2.2, it should be renumbered from I.1 to I.2. Then a new I.1, “Differences between V2.2.2 and V2.2.1” should be created.